src/applications/vaos/appointment-list/redux/reducer.js

```diff
@@ -64,9 +64,6 @@ export default function appointmentsReducer(state = initialState, action) {
     case FETCH_FUTURE_APPOINTMENTS:
       return {
         ...state,
-        pendingStatus: action.includeRequests
-          ? FETCH_STATUS.loading
-          : state.pendingStatus,
```

Comment:
No longer needed since call 'fetchPendingAppointments' will update the state appropriately.